src/vendors/oasisscan.ts

```diff
@@ -53,23 +53,26 @@ export function parseAccount(account: AccountsRow): Account {
 }
 
 export function parseValidatorsList(validators: ValidatorRow[]): Validator[] {
-  return validators.map(v => {
-    const parsed: Validator = {
-      address: v.entity_address,
-      name: v.name ?? undefined,
-      escrow: parseStringValueToInt(v.escrow),
-      current_rate: v.commission,
-      status: v.status ? 'active' : 'inactive',
-      media: {
-        email_address: v.email ?? undefined,
-        logotype: v.icon ?? undefined,
-        twitter_acc: v.twitter ?? undefined,
-        website_link: v.website ?? undefined,
-      },
-      rank: v.rank,
-    }
-    return parsed
-  })
+  return validators
+    .slice()
+    .sort((a, b) => Number(b.status) - Number(a.status))
```

Comment:
This isn't readable. Needs a comment or refactor.

Reply:
I will just add helpers too vendors and use one sort function for both BEs